biorbd_optim/__init__.py

```diff
@@ -20,6 +20,11 @@ class OdeSolver (enum.Enum):
 class Variable:
     @staticmethod
     def variable_torque_driven(nlp):
+        """
+        Catches name from model and gives it to each states and controls.
+        :param nlp: includes all parameters, states, controls, model, objective functions, constraints, limits, etc.
+        :return: nothing, but names states (nlp.x) and controls (nlp.u) and gives size to (nlp.nx) and (nlp.nu)
```

Comment:
param nlp should refer to its actual name `OptimalControlProgram` and not what is in `OptimalControlProgram` (e.g. :param nlp: An OptimalControlProgram class)
If no return, do not put it. However what is explained here should appear in the description (nlp.x, etc.)